### PR TITLE
feat: build date picker ui

### PR DESCRIPTION
**Pull request by raymond-le-goldenowl**
**Task**

Build date picker ui

**Descriptions**

User can select datetime

**Screenshots**
Screenshot from 2022-01-25 17-00-50

### CODE REVIEW

File: src/app/components/DatePickerCustom/index.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	const handleOnchange = value => getValueSelected(moment(value).format(DATE_FORMAT));
+
+	return (
+		<ConfigProvider locale={viVN}>
+			<DatePicker
+				className='date-picker-custom'
+				defaultValue={
+					defaultDateTime
+						? moment(defaultDateTime, DATE_FORMAT)
+						: moment(moment(moment()._d).format(DATE_FORMAT), DATE_FORMAT)
```
Comment (jun-nguyen-goldenowl) [inline src/app/components/DatePickerCustom/index.jsx]: is there any other simple way?

**Review by jun-nguyen-goldenowl — CHANGES_REQUESTED** [review]

File: src/app/components/DatePickerCustom/index.jsx (hunk 1 above)
Comment (raymond-le-goldenowl) [reply src/app/components/DatePickerCustom/index.jsx]: i can use something like this 
```js 
defaultDateTime ? moment(defaultDateTime, DATE_FORMAT) : moment(new Date(), DATE_FORMAT)
```

File: src/app/components/DatePickerCustom/index.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import './styles.scss';
+
+// custom local days on a week and months on a year.
+moment.updateLocale('vn', {
+	weekdaysMin: WEEKS,
+	monthsShort: MONTHS
+});
+
+export default function DatePickerCustom({ defaultDateTime, onChange }) {
+	const handleOnchange = value => onChange && onChange(moment(value).format(DATE_FORMAT));
```
Comment (pwn-0x309) [inline src/app/components/DatePickerCustom/index.jsx]: use optional chaining syntax instead

Comment (pwn-0x309) [reply src/app/components/DatePickerCustom/index.jsx]:
```suggestion
	const handleOnchange = value => onChange?.(moment(value).format(DATE_FORMAT));
```

**Review by pwn-0x309 — CHANGES_REQUESTED** [review]

**Review by pwn-0x309 — APPROVED** [review]

**Pull request merged**